ringbuf/ring.go

```diff
@@ -77,8 +77,10 @@ func (rr *ringReader) isEmpty() bool {
 	return prod == cons
 }
 
+// The data pages in ring buffers are mapped twice in a single contiguous virtual region
+// Therefore the true size is half the size of the mmaped region
```

Comment:
Maybe we can be more explicit here?
```suggestion
// To be able to wrap around data, data pages in ring buffers are mapped twice in
// a single contiguous virtual region.
// Therefore the returned usable  size is half the size of the mmaped region.
```

Reply:
@florianl Usama is on PTO for the next week or so. Do you have permissions to make changes on his branch? This doc change looks good to me, although there is an extra space between `usable` and `size`.